collector/monitors/stubs.go

```diff
@@ -0,0 +1,46 @@
+package monitors
```

Comment:
`stubs.go` contains only initial testing artifacts and should not  build with main code